src/app/components/category-products/category-products.component.html

```diff
@@ -13,6 +13,35 @@ <h5 class="mb-3">Categories</h5>
           </a>
         </li>
       </ul>
+
+      <h5 class="mb-4 mt-8">Price</h5>
+      <input
+        type="text"
+        class="form-control mb-4"
+        placeholder="Price From"
+        [formControl]="priceFrom"
+      />
+      <input
+        type="text"
+        class="form-control mb-4"
+        placeholder="Price To"
+        [formControl]="priceTo"
+      />
+
+      <h5 class="mb-4 mt-8">Rating</h5>
+      <div *ngFor="let item of model.filters.ratingOptions" class="form-check mb-2">
+        <input
```

Comment:
it would be better if the app-rating component already had the clicked or changed event emitter. This way you would not need this input here

Reply:
@gradzio I need input to display currently selected rate, it's required only in filters, so I think event emitter isn't necessary in this case

Reply:
ok